### ISSUE TITLE
Fatal error with WP 4.5 RC 1 and WP-CLI v0.23.0

### ISSUE THREAD

**Issue by NateWr** [wp-cli/wp-cli#2635]
When I try to run a command I get a fatal error. The error seems to occur with any command which bootstraps WordPress. For instance, I can run `wp` and `wp plugin` but not `wp plugin list`.

The fatal error:

```
PHP Warning:  require(/public/htdocs/wp-includes/class-wp-metadata-lazyloader.php): failed to open stream: No such file or directory in phar:///usr/local/bin/wp/php/utils-wp.php on line 60
Warning: require(/public/htdocs/wp-includes/class-wp-metadata-lazyloader.php): failed to open stream: No such file or directory in phar:///usr/local/bin/wp/php/utils-wp.php on line 60
PHP Fatal error:  require(): Failed opening required '/public/htdocs/wp-includes/class-wp-metadata-lazyloader.php' (include_path='phar:///usr/local/bin/wp/vendor/phpunit/phpunit-mock-objects:phar:///usr/local/bin/wp/vendor/phpunit/php-token-stream:phar:///usr/local/bin/wp/vendor/phpunit/php-code-coverage:phar:///usr/local/bin/wp/vendor/phpunit/phpunit:phar:///usr/local/bin/wp/vendor/symfony/yaml:.:/usr/share/php:/usr/share/pear') in phar:///usr/local/bin/wp/php/utils-wp.php on line 60
Fatal error: require(): Failed opening required '/public/htdocs/wp-includes/class-wp-metadata-lazyloader.php' (include_path='phar:///usr/local/bin/wp/vendor/phpunit/phpunit-mock-objects:phar:///usr/local/bin/wp/vendor/phpunit/php-token-stream:phar:///usr/local/bin/wp/vendor/phpunit/php-code-coverage:phar:///usr/local/bin/wp/vendor/phpunit/phpunit:phar:///usr/local/bin/wp/vendor/symfony/yaml:.:/usr/share/php:/usr/share/pear') in phar:///usr/local/bin/wp/php/utils-wp.php on line 60
```

Running WP 4.5 RC 1 with WP-CLI version 0.23.0:

```
admin@mindingkids:~/public/htdocs$ wp --info
PHP binary:	/usr/bin/php5
PHP version:	5.4.45-0+deb7u2
php.ini used:	/etc/php5/cli/php.ini
WP-CLI root dir:	phar://wp-cli.phar
WP-CLI packages dir:	/srv/.wp-cli/packages/
WP-CLI global config:	
WP-CLI project config:	
WP-CLI version:	0.23.0
```

**Comment by danielbachhuber** [comment]
`wp-includes/class-wp-metadata-lazyloader.php` is a new file included in WP 4.5 that doesn't seem present in your RC install. Can you try re-installing WP 4.5 RC 1?

**Comment by NateWr** [comment]
Sorry, my mistake! I must have had one of the early betas running on the site. Thought I was on RC1.
